### PR TITLE
Step 4 Complete! [ CORRECT ]

### PR DESCRIPTION
**Pull request by nisom1**
- Added a comments section (form) on the main page (index.html)
- Takes in the user's comments and places them in an array list

Note: The other branch wasn't up-to-date, I think I was editing on the master branch on accident! So when I went back to my NiaDevelopment branch it had old content.
To solve this problem, I made a new branch (NiaDev) and pulled from the "un-pushed" master branch haha
- I deleted the old branch (NiaDevelopment) because it was old
- ALSO: the command "git push origin BRANCH_NAME" do not put your commits to that BRANCH_NAME. To push your commits to a branch you have to be in/on that branch.
- Learned a lot about GitHub! It's quite neat!

### CODE REVIEW

**Comment by nisom1** [comment]
Wait, I need to make changes

**Comment by nisom1** [comment]
I need to make some more changes before this is merged

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,6 +14,10 @@
```
Comment (SeanCampbell) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Are any of the changes in this file needed, or can they all be reverted?

File: portfolio/src/main/java/com/google/sps/servlets/JSONServlet.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
```
Comment (SeanCampbell) [inline portfolio/src/main/java/com/google/sps/servlets/JSONServlet.java]: And do you still need this file, or can this be deleted in favor of the comment servlet?

**Pull request merged**